OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -250,12 +272,22 @@ public boolean isAnswersEnabled() {
             return mAnswersEnabled;
         }
 
-        public String getFabricKey() {
-            return mFabricKey;
+        public Kit[] getEnabledKits()   {
+            List<Kit> fabricKits = new ArrayList<>();
+
+            if (isCrashlyticsEnabled())    {
+                fabricKits.add(new CrashlyticsCore());
+            }
+
+            if (isAnswersEnabled()) {
+                fabricKits.add(new Answers());
+            }
+
+            return fabricKits.toArray(new Kit[fabricKits.size()]);
         }
 
-        public String getFabricBuildSecret() {
-            return mFabricBuildSecret;
+        public boolean isEnabled()  {
```

Comment:
Might be more clear to rename this to `hasEnabledKits()`, since there isn't actually any specific "enabled" flag, which is what the current name implies.